src/current/v25.2/plpgsql.md

```diff
@@ -351,6 +351,42 @@ BEGIN
 	RETURN;
 ~~~
 
+#### `RETURN NEXT` and `RETURN QUERY`
```

Comment:
We might want to add something to the `RETURN` section above about set-returning functions, since `RETURN` with no parameters serves a similar function there. Are we also missing some docs about using `RETURN` with parameters for scalar functions?

Reply:
Thanks! I went ahead and documented all this. Could you PTAL?